test/controllers/DeparturesControllerSpec.scala

```diff
@@ -574,6 +590,7 @@ class DeparturesControllerSpec extends SpecBase with ScalaCheckPropertyChecks wi
         val request = FakeRequest(GET, routes.DeparturesController.getDepartures().url)
         val result  = route(application, request).value
 
+        contentAsString(result) mustEqual "Failed with the following error: java.lang.Exception"
```

Comment:
We never want to return a string like this. The code should be amended to return a status code only. As this is an internal failure we could log when fetchAllDepartures returns a failed Future.